src/index.ts

```
@@ -164,16 +164,9 @@ export async function getConfig(
+    // init placeholder record for all rpcs
     const rpcRecords: Record<string, RpcRecord> = {};
-    rpcUrls.forEach(
-        (v) =>
-            (rpcRecords[v.endsWith("/") ? v : v + "/"] = {
-                req: 0,
-                success: 0,
-                failure: 0,
-                cache: {},
-            }),
-    );
+    rpcUrls.forEach((v) => (rpcRecords[v.endsWith("/") ? v : v + "/"] = RpcRecord.init()));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)